### PR TITLE
Update license info in FAQ, License and Readme

### PR DESCRIPTION
**Pull request by aurindam**

### CODE REVIEW

File: FAQ.md [hunk 1]
```diff
@@ -51,97 +55,100 @@ types of programming languages.
...
 ## Licensing
 
-You are free to use Slint under either [GNU GPLv3](./LICENSES/GPL-3.0-only.txt) or 
-a [commercial license](./LICENSES/LicenseRef-Slint-commercial.md). We also offer a completely 
-free-of-charge commercial license under the [ambassador program](https://slint-ui.com/ambassador-program.html).
+Slint is available under either a [proprietary license](LICENSES/LicenseRef-Slint-commercial.md)
+or [GNU GPLv3](LICENSES/GPL-3.0-only.txt)
 
-### What are the commercial licensing options?
+### Can I license my code under a more permissive license than GPL?
```
Comment (ogoffart) [inline FAQ.md:61]:
```suggestion
### Can I license my code under a open-source license more permissive than GPL?
```

File: FAQ.md [hunk 2]
```diff
@@ -51,97 +55,100 @@ types of programming languages.
...
 
-We offer - Ambassador, Flex and Buyout commercial licensing options. All three 
-options include a perpetual right to use Slint to develop applications, unlimited 
-distribution of such applications on desktops and [standard support](https://slint-ui.com/support/slint_support_service_agreement).
+Yes. You can publish your own source code under a permissive license compatible with
+the GPL, such as BSD, MIT, or Apache license. However, the binary or the package
+containing Slint needs to be licensed under GPL. For more info, you can check the following GPL FAQs:
```
Comment (ogoffart) [inline FAQ.md:65]:
```suggestion
Yes. While the binary or the package containing Slint needs to be licensed under GPL,
the code you write can be under any license compatible with the GPL, such as BSD, MIT, or Apache license. 
For more info, you can check the following GPL FAQs:
```

File: FAQ.md [hunk 3]
```diff
@@ -51,97 +55,100 @@ types of programming languages.
...
 
-- The Ambassador option is a free license that can be provided in lieu of helping
-us promote Slint. Apart from the standard inclusiosn, unlimited User-Seats, updates and unlimited distribution 
-of Slint based applications on embedded devices are included.
+- [https://www.gnu.org/licenses/gpl-faq.en.html#LinkingWithGPL](https://www.gnu.org/licenses/gpl-faq.en.html#LinkingWithGPL)
+- [https://www.gnu.org/licenses/gpl-faq.en.html#IfLibraryIsGPL](https://www.gnu.org/licenses/gpl-faq.en.html#IfLibraryIsGPL)
```
Comment (ogoffart) [inline FAQ.md:68]:
```suggestion
- [https://www.gnu.org/licenses/gpl-faq.en.html#IfLibraryIsGPL](https://www.gnu.org/licenses/gpl-faq.en.html#IfLibraryIsGPL)

In other words, you can still allow others to fork your code for proprietary software.
The person doing that have the choice to either use Slint according to one of our proprietary options, or remove the dependency.
```

File: FAQ.md [hunk 4]
```diff
@@ -51,97 +55,100 @@ types of programming languages.
...
 
-- With the Flex option, you can purchase the required number of User-Seats needed to
-develop your applications with Slint. In addition to the standard inclusions, updates are included as long 
-as the subscription is active.
+### What are the different proprietary licensing options?
```
Comment (ogoffart) [inline FAQ.md:70]: I wouldn't put the pricing in the FAQ in the repository, just a link to ir.

File: FAQ.md [hunk 5]
```diff
@@ -51,97 +55,100 @@ types of programming languages.
...
-The Ambassador is really for free! The only 'catch' is to help us promote Slint in the following ways:
+### What does perpetual mean?
 
-- Showcase: You grant us the right to use your name, logo and your Products in our marketing materials,
+The perpetual right allows you to use the version(s) of Slint, provided under the
+proprietary license, for ever.
 
-- Spread the word: You will include reasonably visible notices in the documentation, marketing materials 
-and social media channels related to your Products that such were built with Slint,
+### What are the different support options?
```
Comment (ogoffart) [inline FAQ.md:90]: Same, i'd just link to the website.

File: FAQ.md (hunk 4 above)
Comment (aurindam) [reply FAQ.md:73]: Replaced with link

File: FAQ.md (hunk 5 above)
Comment (aurindam) [reply FAQ.md:93]: Since we do not have a comparison table, removed it for now and instead mentioned - Standard support and premium support.

**Pull request merged**